ios/Capacitor/Capacitor/CAPInstanceConfiguration.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+extension InstanceConfiguration {
+    @objc public func getPluginConfigValue(_ pluginId: String, _ configKey: String) -> Any? {
+        return (pluginConfigurations as? JSObject)?[keyPath: KeyPath("\(pluginId).\(configKey)")]
```

Comment:
How come you can't do this like in other places?

```suggestion
        return (pluginConfigurations as? JSObject)?[keyPath: "\(pluginId).\(configKey)"]
```

Reply:
Because this is not using a string literal (which is fully defined at compile time) like "server.url"; this string is assembled from variables at runtime. The other way works through some implicit type conversions to create a KeyPath from the literal automatically.

I did consider making a second subscript operator for `JSObject` that accepted a string and converted it into a `KeyPath` but deferred to being more explicit. That felt like it was getting too clever for the sake of syntactic sugar but I don't feel that strongly about it either.